gammapy/catalog/fermi.py

```diff
@@ -217,7 +217,7 @@ class SourceCatalogObject4FGL(SourceCatalogObjectFermiBase):
         "ASSOC_GAM2",
         "ASSOC_GAM3",
     ]
-    _ebounds = u.Quantity([50, 100, 300, 1000, 3000, 10000, 30000, 300000], "MeV")
+    energy_edges = u.Quantity([50, 100, 300, 1000, 3000, 10000, 30000, 300000], "MeV")
```

Comment:
Name is OK now, but please leave it as a hidden attribute with a trailing `_`...like `_energy_edges`